### PR TITLE
Add --mapping option to upload command

### PR DESCRIPTION
**Pull request by Leland-Takamine**

### CODE REVIEW

File: maestro-cli/src/main/java/maestro/cli/command/UploadCommand.kt [hunk 1]
```diff
@@ -48,34 +48,37 @@ import kotlin.system.exitProcess
...
-    @Option(names = ["--apiKey"])
+    @Option(order = 0, names = ["--apiKey"], description = ["API key"])
     private var apiKey: String? = null
 
-    @Option(names = ["--apiUrl"])
+    @Option(order = 1, names = ["--apiUrl"], description = ["API base URL"])
     private var apiUrl: String = "https://api.mobile.dev"
 
-    @Option(names = ["--repoOwner"])
+    @Option(order = 1, names = ["--mapping"], description = ["(Android-only) Proguard mapping file"])
```
Comment (amanjeetsingh150) [inline maestro-cli/src/main/java/maestro/cli/command/UploadCommand.kt:66]: We also support dsym file which will be important for memory leak, and crash symbolication. I think let's include that as well.

File: maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt [hunk 2]
```diff
@@ -103,6 +104,7 @@ class ApiClient(
             .setType(MultipartBody.FORM)
             .addFormDataPart("app_binary", "app.zip", appFile.toFile().asRequestBody("application/zip".toMediaType()).observable(progressListener))
             .addFormDataPart("workspace", "workspace.zip", workspaceZip.toFile().asRequestBody("application/zip".toMediaType()))
+            .addFormDataPart("mapping", "mapping.txt", workspaceZip.toFile().asRequestBody("text/plain".toMediaType()))
```
Comment (amanjeetsingh150) [inline maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt:107]: @artem888 The dsym of iOS are upload with same key right?

File: maestro-cli/src/main/java/maestro/cli/command/UploadCommand.kt (hunk 1 above)
Comment (Leland-Takamine) [reply maestro-cli/src/main/java/maestro/cli/command/UploadCommand.kt:66]: Ah yep good catch

File: maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt (hunk 2 above)
Comment (Leland-Takamine) [reply maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt]: Yes I believe so

**Comment by Leland-Takamine** [comment]
@amanjeetsingh150 Addressed comments

File: maestro-cli/src/main/java/maestro/cli/command/UploadCommand.kt [hunk 3]
```diff
@@ -48,34 +48,37 @@ import kotlin.system.exitProcess
...
-    @Option(names = ["--branch"])
+    @Option(order = 5, names = ["--branch"], description = ["The branch this upload originated from"])
     private var branch: String? = null
 
-    @Option(names = ["--pullRequestId"])
+    @Option(order = 6, names = ["--pullRequestId"], description = ["The ID of the pull request this upload originated from"])
     private var pullRequestId: String? = null
 
-    @Option(names = ["-e", "--env"])
+    @Option(order = 6, names = ["-e", "--env"], description = ["Environment variables to inject into your Flows"])
```
Comment (amanjeetsingh150) [inline maestro-cli/src/main/java/maestro/cli/command/UploadCommand.kt:81]: Shouldn't this order be 7?

Comment (Leland-Takamine) [reply maestro-cli/src/main/java/maestro/cli/command/UploadCommand.kt]: Oops yes - updating!

**Review by amanjeetsingh150 — APPROVED** [review]

**Review by dmitry-zaitsev — APPROVED** [review]

File: maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt (hunk 2 above)
Comment (artem888) [reply maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt]: yes , it is

**Pull request merged**